templates/releases/build_queue.html

```diff
@@ -84,9 +84,40 @@
                         </li>
                     {%- endfor %}
                 {%- else %}
-                    <strong>There is nothing in the queue</strong>
+                    <strong>There is nothing in the build queue</strong>
                 {%- endif %}
             </ol>
+
+            <div class="release">
+                <strong>Rebuild Queue</strong>
+            </div>
+
+            <div class="about">
+                <p>
+                    We continuously rebuild the latest versions for all crates so they can
+                    benefit from new features in rustdoc.
+                </p>
+                {%- if !expand_rebuild_queue -%}
+                    <p>There are currently {{ rebuild_queue.len() }} crates in the rebuild queue.</p>
```

Comment:
Please use the [`pluralize`](https://rinja.readthedocs.io/en/stable/filters.html#pluralize) filter on `crates`.